recipe/meta.yaml

```diff
@@ -20,18 +20,17 @@ requirements:
     - python                                 # [build_platform != target_platform]
   host:
     - python
-    - pip >=18
-    - setuptools >=41
+    - pip
+    - setuptools >=41.0.0
     - setuptools_scm >=2
```

Comment:
For v20.16.5:
```
    "setuptools>=59.6",
    "setuptools-scm>=6.4.2"
```
But here are contradicting constraints https://github.com/pypa/virtualenv/blob/20.16.5/setup.py. I guess it's not updating anymore

Reply:
It looks an oversight by the developers. And since the constraints are not directly contradicting, I assume nobody noticed.